marketplace/deployer_util/provision.py

```diff
@@ -53,6 +53,16 @@ def process(schema, values, deployer_image, deployer_entrypoint):
   app_name = get_name(schema, values)
   namespace = get_namespace(schema, values)
 
+  # Inject DEPLOYER_IMAGE property values if not already present.
+  for key, value in values.items():
+    if value:
+      continue
+    if key not in schema.properties:
+      continue
+    if not schema.properties[key].xtype == 'DEPLOYER_IMAGE':
+      continue
+    values[key] = deployer_image
```

Comment:
can you extract this into a function and add a test case? This file is missing a lot of tests, but I think we can test this particular thing.

Reply:
Added a test, though I'm unsatisfied with it.

I'd be happy with a test that validated that `schema.yaml` + `values.yaml` -> `manifest.yaml`, but there's other higher-impact broader-scope tests that would add more value (e.g. sample deployers for each of the base deployers).